config-connector/solutions/iam/kpt/pubsub-subscription/README.md

```diff
@@ -0,0 +1,44 @@
+pubsub-subscription
+==================================================
+
+# NAME
+
+  pubsub-subscription
+
+# SYNOPSIS
+
+  This package configures permissions for a pubsub subscription by creating an IAMPolicyMember resource
+
+# CONSUMPTION
+
+  Using [kpt](https://googlecontainertools.github.io/kpt/):
+
+  ```
+  SRC=https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit.git
+  kpt pkg get $SRC/cloud-foundation-toolkit.git/config-connector/solutions/iam/kpt/pubsub-subscription .
+  ```
+
+# REQUIREMENTS
+
+  A working cluster with Config Connector installed managing a GCP project with PubSub API enabled. 
```

Comment:
This sounds like a cluster is managing a GCP project and I think this doesn't really describe the relationship properly.

Maybe something like "Config Connector successfully installed in a K8s cluster and the project managed by Config Connector has Pub/Sub API enabled"?